diff --git a/node_modules/@shopify/flash-list/RNFlashList.podspec b/node_modules/@shopify/flash-list/RNFlashList.podspec
index 1359794..ca12d9f 100644
--- a/node_modules/@shopify/flash-list/RNFlashList.podspec
+++ b/node_modules/@shopify/flash-list/RNFlashList.podspec
@@ -9,7 +9,7 @@ Pod::Spec.new do |s|
   s.homepage         = package['homepage']
   s.license          = package['license']
   s.author           = package['author']
-  s.platform         = :ios, '11.0'
+  s.platforms        = { :ios => '12.0', :tvos => '12.0' }
   s.source           = { git: 'https://github.com/shopify/flash-list.git', tag: 'v#{s.version}' }
   s.source_files     = 'ios/Sources/**/*'
   s.requires_arc     = true
